scripts/mmvec

```diff
@@ -96,14 +96,18 @@ def mmvec():
               help=('Path to save the ordination learned from the model. '
                     'If this is not specified, then this will be saved under '
                     '`--summary-dir`.'))
+@click.option("--equalize-biplot", default=False, required=False,
```

Comment:
```suggestion
@click.option("--equalize-biplot", default=False, required=False, is_flag=True,
```
This allows the user to just specify the "--equalize-biplot" option as boolean flag.   So if this isn't specified, the equalize_biplot won't be called. If `is_flag` isn't enabled, the user will have to add in text and this command will throw an error..

This is what was in the PR
![image](https://user-images.githubusercontent.com/4184797/71732689-e33f7880-2e15-11ea-947f-4c11389362d6.png)

This is what the `is_flag` option will allow
![image](https://user-images.githubusercontent.com/4184797/71732695-ecc8e080-2e15-11ea-8130-f02cbcd97e89.png)